README.adoc

```diff
@@ -130,6 +130,19 @@ assert(collection.includes?(actual))
 assert_includes(collection, actual)
 ----
 
+=== Refute Includes [[refute-includes]]
+
+Use `refute_includes` if the actual value is not included in the collection.
+
+[source,ruby]
+----
+#bad
+refute(collection.includes?(actual))
```

Comment:
I guess there should also be an example of a negated `assert/assert_includes` as well.

Reply:
Make sense. I'll add that today.

Reply:
Fixed it. Please have a look.